maven-artifact/src/main/java/org/apache/maven/artifact/handler/ArtifactHandler.java

```diff
@@ -41,7 +42,7 @@ public interface ArtifactHandler {
     String getDirectory();
 
     /**
-     * Get the classifier associated to the dependency type.
+     * Returns the classifier of the dependency.
```

Comment:
while we are at clarifying, it is the **default** classifier: user may override the value in his pom.xml dependency (by defining a classifier)

Reply:
so this is the classifier from the artifact's own pom.xml?

Reply:
The `ArtifactHandler` is unrelated to the artifact's own pom.xml.
The `ArtifactHandler` is used to determine a few informations about a dependency.  It is retrieved based on the [_dependency type_](https://maven.apache.org/ref/4-LATEST/apidocs/org/apache/maven/api/model/Dependency.html#getType()).  But the [`Dependency`](https://maven.apache.org/ref/4-LATEST/apidocs/org/apache/maven/api/model/Dependency.html) can define a classifier, which takes precedence over the default one provided by the artifact handler.

Reply:
Agreed, `classifier` -> `default classifier`